static/styles/zulip.css

```diff
@@ -1395,7 +1398,7 @@ td.pointer {
 
 .unread-marker-fill {
     background-color: hsl(107, 74%, 29%);
-    width: 3px;
+    width: 2px;
```

Comment:
Making unread marker thinner to leave more space for the new focus rectangle, and also making it thinner as per the specs.